src/main/java/org/aarboard/nextcloud/api/webdav/Files.java

```diff
@@ -248,25 +237,25 @@ public ResourceProperties getProperties(String remotePath, boolean allProperties
             Set<QName> props= new HashSet<>();
             if (allProperties)
             {
-                props.add(new QName("DAV:", "getlastmodified", "d"));
-                props.add(new QName("DAV:", "getetag", "d"));
-                props.add(new QName("DAV:", "getcontenttype", "d"));
-                props.add(new QName("DAV:", "resourcetype", "d"));
-                props.add(new QName("DAV:", "getcontentlength", "d"));
-                props.add(new QName("DAV:", "displayname", "d"));
-                props.add(new QName("http://owncloud.org/ns", "id", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "fileid", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "favorite", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "comments-href", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "comments-count", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "comments-unread", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "owner-id", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "owner-display-name", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "share-types", "oc"));
-                props.add(new QName("http://owncloud.org/ns", "checksums", "oc"));
+                props.add(new QName(URI_NS_DAV, "getlastmodified", "d"));
+                props.add(new QName(URI_NS_DAV, "getetag", "d"));
+                props.add(new QName(URI_NS_DAV, "getcontenttype", "d"));
+                props.add(new QName(URI_NS_DAV, "resourcetype", "d"));
+                props.add(new QName(URI_NS_DAV, "getcontentlength", "d"));
+                props.add(new QName(URI_NS_DAV, "displayname", "d"));
+                props.add(new QName(URI_NS_OWNCLOUD, "id", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "fileid", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "favorite", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "comments-href", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "comments-count", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "comments-unread", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "owner-id", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "owner-display-name", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "share-types", "oc"));
+                props.add(new QName(URI_NS_OWNCLOUD, "checksums", "oc"));
                 props.add(new QName("http://nextcloud.org/ns", "has-preview", "nc"));
```

Comment:
wasn't sure why this NS is different

Reply:
Thanks for all your work.
I will review it in the next 1-2 weeks (I'm quite busy at the moment)

It's correct, that the `has-preview`property has another namespace.
All other xml elements are comming over from owncloud, where as the `has-preview` is a nextcloud exclusive thing.
https://docs.nextcloud.com/server/latest/developer_manual/client_apis/WebDAV/basic.html